container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -112,6 +112,9 @@ const (
 	successfulExitCode = 0
 
 	emptyNetworkAlias = ""
+
+	// setting this to true to enable "satrian" on Discord to do his workflow
```

Comment:
I think we should leave this comment out of the code base, but other wise good with the change